tools/add_copyright.py

```diff
@@ -259,6 +259,8 @@ def add_copyrights(paths):
 
     # Don't automatically 'git add' changes for now, make it more clear which
     # files were changed and have ability to see 'git diff' on them.
+    # Note that this means the hook will modify files and then cancel the commit, which you will then
+    # have to manually make again.
```

Comment:
>  which you will then have to manually make again.

Can you clarify this? Meaning you'll have to run `git add` manually? Or something else?

Reply:
Yes, you need to do `git add` then `git commit` again if files were modified.